src/Interface/Character/Parses.js

```diff
@@ -544,4 +570,13 @@ class Parses extends React.Component {
   }
 }
 
-export default Parses;
+const mapStateToProps = state => {
+  return ({ });
+};
+
+export default withRouter(connect(
```

Comment:
When using multiple decorators please use the `compose` function from the `redux` library.